METADATA
last updated: 2025-12-16 BA updated metadata after BA fixed inconsistencies
file_name: 2021-03-22_EUA Submission - FloodLAMP QuickColor COVID-19 Test v1.0.md
file_date: 2021-03-22
title: EUA Submission - FloodLAMP QuickColor COVID-19 Test v1.0
category: regulatory
subcategory: fl-fda-submissions
tags: 
source_file_type: docx
xfile_type: docx
gfile_url: https://docs.google.com/document/d/19VRwuRtGafiCtB7rHAn264BqKSJgqEjb
xfile_github_download_url: https://raw.githubusercontent.com/FocusOnFoundationsNonprofit/floodlamp-archive-wip/main/regulatory/fl-fda-subs/2021-03-22_EUA%20Submission%20-%20FloodLAMP%20QuickColor%20COVID-19%20Test%20v1.0.docx
pdf_gdrive_url: https://drive.google.com/file/d/1sS6RlG2aLsXouqmliuSLbQOr4C4eVz2S
pdf_github_url: https://github.com/FocusOnFoundationsNonprofit/floodlamp-archive-wip/blob/main/regulatory/fl-fda-subs/2021-03-22_EUA%20Submission%20-%20FloodLAMP%20QuickColor%20COVID-19%20Test%20v1.0.pdf
conversion_input_file_type: docx
conversion: pandoc
license: CC BY 4.0 - https://creativecommons.org/licenses/by/4.0/
tokens: 10866
words: 6495
notes: 
summary_short: The EUA Submission for the FloodLAMP QuickColor COVID-19 Test v1.0 describes an extraction-free, colorimetric RT-LAMP assay for qualitative SARS-CoV-2 detection using a visual pink-to-yellow readout and no specialized instrumentation. It details intended use for weekly screening of symptomatic and asymptomatic individuals, primer design targeting three viral genes, sample inactivation workflow, controls, and interpretation criteria for CLIA high-complexity labs. It reports analytical and clinical performance including a 12,500 copies/mL LoD and 90% positive agreement with 100% negative agreement in an 80-specimen clinical evaluation.


CONTENT

***INTERNAL TITLE:*** EUA Submission - FloodLAMP QuickColor(TM) COVID-19 Test
## A. PURPOSE FOR SUBMISSION
Emergency Use Authorization (EUA) request for distribution and/or use of the **FloodLAMP QuickColor(TM) COVID-19 Test** for the *in vitro* qualitative detection of RNA from the SARS-CoV-2 in in upper respiratory specimens including oropharyngeal and nasopharyngeal swabs, anterior nasal and mid-turbinate nasal swabs **from individuals suspected of COVID-19 by their healthcare provider and from individuals without symptoms or other epidemiological reasons to suspect COVID-19 infection, when tested at a weekly interval with no more than 9 days between tests.** Additional testing and confirmation procedures should be performed in consultation with public health and/or other authorities to whom reporting is required. Test results should be reported in accordance with local, state, and federal regulations.

## B. MEASURAND
Specific nucleic acid sequences from the genome of the SARS-CoV-2, targeted by **primers from the ORF1ab, N and E regions** of the virus. Primer names and sequences are listed in Table 1.

#### Table 1: Primer names and sequences
|  |  |
|----------------------|--------------------------------------------------|
| **Primer Name** | **Sequence (5'-3')** |
| **ORF1ab gene (AS1e)** |  |
| Orf1ab\_FIP | TCAGCACACAAAGCCAAAAATTTATTTTTCTGTGCAAAGGAAATTAAGGAG |
| Orf1ab\_BIP | TATTGGTGGAGCTAAACTTAAAGCCTTTTCTGTACAATCCCTTTGAGTG |
| Orf1ab\_F3 | CGGTGGACAAATTGTCAC |
| Orf1ab\_B3 | CTTCTCTGGATTTAACACACTT |
| Orf1ab\_LF | TTACAAGCTTAAAGAATGTCTGAACACT |
| Orf1ab\_LB | TTGAATTTAGGTGAAACATTTGTCACG |
| **N Gene (N2)** |  |
| N2\_FIP | TTCCGAAGAACGCTGAAGCGGAACTGATTACAAACATTGGCC |
| N2\_BIP | CGCATTGGCATGGAAGTCACAATTTGATGGCACCTGTGTA |
| N2\_F3 | ACCAGGAACTAATCAGACAAG |
| N2\_B3 | GACTTGATCTTTGAAATTTGGATCT |
| N2\_LF | GGGGGCAAATTGTGCAATTTG |
| N2\_LB | CTTCGGGAACGTGGTTGACC |
| **E Gene (E1)** |  |
| E1\_FIP | ACCACGAAAGCAAGAAAAAGAAGTTCGTTTCGGAAGAGACAG |
| E1\_BIP | TTGCTAGTTACACTAGCCATCCTTAGGTTTTACAAGACTCACGT |
| E1\_F3 | TGAGTACGAACTTATGTACTCAT |
| E1\_B3 | TTCAGATTTTTAACACGAGAGT |
| E1\_LF | CGCTATTAACTATTAACG |
| E1\_LB | GCGCTTCGATTGTGTGCGT |
||

## C. APPLICANT
FloodLAMP Biotechnologies, a DE Public Benefit Corporation
| Mailing Address: 4860 Alpine Rd. Portola Valley, CA 94028 | Laboratory Address: 930 Brittan Ave San Carlos, CA 94070	 | Randall J. True, CSO Phone: (415) 269-2974 Email: randy@floodlamp.bio |
| :---- | :---- | :---- |

## D. PROPRIETARY AND ESTABLISHED NAMES
Proprietary Name - **FloodLAMP QuickColor(TM) COVID-19 Test**
Established Name - **FloodLAMP QuickColor(TM) COVID-19 Test**

## E. REGULATORY INFORMATION
***Approval/Clearance Status:***

The **FloodLAMP QuickColor(TM) COVID-19 Test** is not cleared, CLIA waived, approved, or subject to an approved investigational device exemption.

***Product Code:***

QJR

## F. PROPOSED INTENDED USE
### 1) Intended Use:
**FloodLAMP QuickColor(TM) COVID-19 Test** is a reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) assay intended for the qualitative detection of RNA from SARS-CoV-2 in upper respiratory specimens including nasopharyngeal swabs, anterior nasal and mid-turbinate nasal swabs **from individuals suspected of COVID-19 by their healthcare provider and from individuals without symptoms or other epidemiological reasons to suspect COVID-19 infection, when tested at a weekly interval with no more than 9 days between tests.** Testing is limited to **laboratories certified under the Clinical Laboratory Improvement Amendments of 1988 (CLIA), 42 U.S.C. §263a, to perform high complexity tests, or by similarly qualified non-U.S. laboratories.**

Results are for the identification of SARS-CoV-2 RNA. The SARS-CoV-2 RNA is generally detectable in upper respiratory specimens including nasopharyngeal swabs, anterior nasal and mid-turbinate nasal swabs during the acute phase of infection. Positive results are indicative of the presence of SARS-CoV-2 RNA; clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease. Laboratories within the United States and its territories are required to report all test results to the appropriate public health authorities.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basis for patient management decisions. Negative results must be combined with clinical observations, patient history, and epidemiological information.

The **FloodLAMP QuickColor(TM) COVID-19 Test** is intended for use by **qualified and trained clinical laboratory personnel specifically instructed and trained in the techniques of *in vitro* diagnostic procedures**. The **FloodLAMP QuickColor(TM) COVID-19 Test** is only for use under the Food and Drug Administration's Emergency Use Authorization.

### 2) Special Conditions for Use Statements:
For Emergency Use Authorization (EUA) only
For prescription use only
For *in vitro* diagnostic use only

### 3) Special Instrument Requirements:
The **FloodLAMP QuickColor(TM) COVID-19 Test** does not have special instrument requirements.

## G. DEVICE DESCRIPTION AND TEST PRINCIPLE
### 1) Product Overview/Test Principle:
The **FloodLAMP QuickColor(TM) COVID-19 Test** is a RNA extraction-free reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) molecular assay that indicates the presence of the SARS-CoV-2 viral RNA with a simple visual color change. It can widely and rapidly scale because 1) no special instrumentation of any kind is required, neither nucleic acid extraction equipment nor a RT-PCR instrument, 2) it utilizes reagents and supplies readily available in large quantities, and 3) is a very straightforward protocol with minimal steps that can be executed quickly and reliably. It also utilizes the same streamlined sample preparation as the **FloodLAMP EasyPCR(TM) COVID-19 Test**. Both are supply chain robust, "open source" protocol tests, meaning designated laboratories may obtain the test components directly from vendors. Together, the two tests can be used in an integrated program for screening and rapid confirmation in large populations by a broad range of laboratories.

The **FloodLAMP QuickColor(TM) COVID-19 Test** uses a set of specific primers that target ORF1ab, N and E genes for the detection of SARS-CoV-2 RNA. It uses Loop Mediated Isothermal Amplification (LAMP), a nucleic acid amplification technique wherein DNA amplification is carried out at a constant temperature of approximately 65°C. Samples are first treated with a TCEP-based Inactivation Solution followed by a heat inactivation step. The resulting inactivated sample is directly used as input in the LAMP reaction. The amplification reaction mix includes a reverse transcriptase (RT) polymerase to create complementary cDNA from RNA. The cDNA is subsequently amplified by a high strand displacement DNA polymerase. The amplified DNA products lower the pH of the reaction. A phenol red pH indicating dye is included in the amplification reaction mix, thus causing the reaction solution to visibly change from an initial bright pink to a bright yellow when sufficient amplification occurs. Reactions that change color to yellow indicate that SARS-CoV-2 RNA is present.

### 2) Description of Test Steps:
Specimens including **nasopharyngeal swabs, anterior nasal and mid-turbinate nasal swabs** are collected in **sterile collection tubes.** Swabs are transported and stored dry prior to processing. At the laboratory, an inactivation solution at 1X containing TCEP (2.5 mM), EDTA (1 mM), and NaOH (11 mM) in 0.9% saline (154 mM) is added to the container with the swab, at the volume of 1 mL. Alternatively, for swabs that are collected or eluted in a saline solution or equivalent, the inactivation solution at 100X concentration should be added at 1/100th the sample solution volume.

The container with the specimen and inactivation solution is mixed by vortexing for 30 seconds. Subsequently, the container is heated for 8 minutes in a 95°C water bath or dry heat block. The now inactivated specimen container is allowed to cool at room temperature for 10 minutes and then stored on ice or at 4°C until amplification.

An amplification reaction mix (23 µl) is prepared per manufacturer's specifications, containing the Colorimetric LAMP master mix (NEB M1800, 12.5 µl) and a primer-guanidine solution (10.5 µl) comprising 10X primers mix (2.5 µl of 10X w FIP/BIP at 16 µM, F3/B3 at 2 µM, and LF/BF at 4 µM), guanidine hydrochloride (2.5 µl of 400 mM), and nuclease-free water (5.5 µl). The primer-guanidine solution may be prepared ahead of time and stored at -20°C for up to 1 month.

2 µl of the inactivated sample is added to 23µl of the amplification reaction mix. The reaction is incubated at 65°C for 25 minutes in a thermal cycler, dry heat block, or water bath. After removal from the heat, the reaction solution is allowed to cool at room temperature for 1 minute and then the test result is determined visually based on the color of the reaction solution.

### 3) Control Materials to be Used with test:
**One positive and one negative control** will be included on every 96-well plate with up to 94 samples, or with every batch of strip tubes on each heater:

1.  A "no template" (negative) control (NTC) is needed to **assure the absence of cross contamination from positive samples, positive controls, or amplicons** and is used **to determine if sample results are valid.** **It consists of 100X Inactivation Solution diluted to 1X in 0.9% saline.** This NTC is the same solution added to dry swabs (see Section I below for the components).

2.  A positive template control is needed to **assure proper functioning of reagents and the absence of significant RNAse contamination. It consists of synthetic viral RNA at a concentration of approximately 100,000 cp/mL diluted in total human RNA and nuclease-free water.** Stock and working aliquots of the positive control are produced from the sources listed in Table 2 or equivalents. Working aliquots should be diluted prior to use to 100,000 cp/mL. Positive control aliquots should be stored for at most 3 months at -80°C, or at most 1 month at -20°C.

#### Table 2: Components for Positive Template Control
|                                |               |                |            |
|--------------------------------|---------------|----------------|------------|
| **Material**                   | **Supplier**  | **Catalog \#** | **Volume** |
| SARS-CoV2 Positive Control RNA | Twist         | 102019         | 5 µL       |
| Total Human RNA                | Thermo Fisher | 4307281        | 100 µL     |
| Nuclease-free Water            | Thermo Fisher | 10977015       | 4,895 µL   |
||

## H. INTERPRETATION OF RESULTS
### 1) FloodLAMP QuickColor(TM) COVID-19 Test Controls
All test controls should be examined prior to interpretation of specimen results. If the controls are not valid, the specimen results cannot be interpreted. An example of the expected appearance of the negative and positive controls after amplification is shown in Figure 1.

![][image1]

#### Figure 1. Negative control (left) and positive control (right) after amplification.
If the negative and positive controls do not appear as expected, the specimen results of the corresponding plate or batch should be considered invalid. In the event of a failure of either the positive or negative control, the lab should discard some or all of the consumables utilized for associated run, including the filter tips, tubes, plates, seals, and aliquots of reagents. Additionally, all pipettes, BSC, and appropriate lab surfaces should be thoroughly cleaned with freshly made 10% bleach solution, 70% ethanol, and optionally RNAseZAP product. In the event of the failure of the positive control, the working aliquot of positive control material should be discarded. Additionally, the lab should review the expiration of the batch of positive control aliquots and verify their integrity by performing qualification reactions of one or more positive control aliquots. If controls continue to fail, labs should not perform additional tests on clinical specimens or report results. Invalid test results should be repeated by performing another amplification reaction.

### 2) Examination and Interpretation of Patient Specimen Results:
Assessment of clinical patient specimen test results should be performed after the positive and negative controls have been examined and determined to be valid and acceptable. If the controls are not valid, the patient results cannot be interpreted.

Test results should be read at least 1 minute and no more than 8 hours after plates or tubes have been removed from heat. Test results may be determined directly from visual inspection of the color of the reaction tubes:

- yellow - result is positive  
- bright pink or red - result is negative  
- any other color - result is inconclusive.

Examples are shown below in Figure 2. Edge cases for positive and negative results are shown below in Figure 3. Any color variance stronger than the edge cases should be interpreted as inconclusive. In order to reduce the chance of both false negative and false positive results, this window for color variance is intentionally set to be small.

If the initial test is inconclusive, then one of the following should be performed:

1) repeat the Colorimetric LAMP Amplification Reaction on the inactivated sample. If the repeat test has a positive result then the final interpretation of the test is positive. If the repeat test has a negative or another inconclusive result, then the final interpretation is inconclusive.

2) follow-up test the inactivated sample with the FloodLAMP EasyPCR(TM) COVID-19 Test or another high sensitivity EUA authorized test that comprises the same inactivation protocol. The final interpretation is the result of the follow-up test.

For serial screening of individuals without symptoms or other epidemiological reasons to suspect COVID-19 infection, the initial inconclusive test result may be considered the final interpretation. If the final interpretation of the test result is inconclusive, then "Inconclusive" should be reported and retesting of the individual is recommended.

![][image2]

## I. PRODUCT MANUFACTURING
### 1) Overview of Manufacturing and Distribution:
The **FloodLAMP QuickColor(TM) COVID-19 Test** utilizes standard chemicals available in very large quantities from multiple vendors, with the exception of the LAMP Primers and New England Biolabs Colorimetric LAMP Master Mix.

FloodLAMP has partnered with LGC Biosearch for the LAMP Primers. LGC Biosearch has very large scale oligo production capacity and mature distribution capabilities. The first production scale lot of the LAMP Primers has been completed, with 1.2 million reactions ready for immediate distribution. FloodLAMP has purchased 600K reactions of the LAMP Primers. LGC Biosearch is supplying FloodLAMP for the **FloodLAMP QuickColor(TM) COVID-19 Test** and is also offering the LAMP Primers as a catalog product.

New England Biolabs has expressed strong support for the **FloodLAMP QuickColor(TM) COVID-19 Test** and FloodLAMP's other open source protocol EUA submissions that incorporate their products. New England Biolabs has very large quantities of the Colorimetric LAMP Master Mix product prepared and ready for immediate distribution, typically with 24 hour shipping within the U.S. Their manufacturing capacity is among the largest in the United States and can surge to meet increased demand.

**\*Under the Emergency Use Authorization (EUA) any of the 21 CFR Part 820 Quality System Regulation (QSR) requirements can be waived for the duration of the EUA but FDA recommends that developers follow comparable practices as much as possible if such requirements are waived. Among other things, FDA may consider previous compliance history when determining whether or not to waive certain QSR requirements for a specific product. Please note adverse events, as per 21 CFR Part 803, have to be reported for authorized devices (see Section P).**

### 2) Components Included with the Test
None. Designated CLIA labs may order components directly from vendors.

### 3) Components Required But Not Included with Test:
The **FloodLAMP QuickColor(TM) COVID-19 Test** is to be used with the reagents or equivalents listed in Table 3. No specialized instruments are needed. Only ordinary laboratory equipment such as pipettes, centrifuges, and heaters are needed.

#### Table 3: Validated reagents used with Test
| Item                    | Concentration | Chemical Composition                                 | Vendor                | Catalog No.          |
| :---------------------- | :-----------: | :---------------------------------------------------- | :-------------------- | :-------------------- |
| TCEP                    | .5 M         | tris(2-carboxyethyl)phosphine hydrochloride          | Sigma-Aldrich Millipore Sigma | 646547-10X1ML |
| EDTA                    | .5 M         | Ethylenediaminetetraacetic acid                      | Thermo Fisher         | 15575020             |
| NaOH                    | 10 N         | Sodium Hydroxide                                     | Sigma-Aldrich        | SX0607N-6            |
| Nuclease- free Water    |       -       | Ultrapure Water, nuclease-free                       | Thermo Fisher         | 10977015             |
| NaCl                    | 5 M          | Sodium Chloride                                      | Thermo Fisher         | 24740011             |
| Guanidine               | 6 M          | Guanidine Hydrochloride                              | Sigma-Aldrich        | SRE0066              |
| ColorimetricLAMP MM*    |       -       | Colorimetric LAMP Master Mix                         | New England Biolabs   | M1804                |
||

\* Item may not be substituted for equivalents. Only the specified vendor and catalog number may be utilized.

Stocks of TCEP, EDTA, NaOH, and NaCl may be prepared from powder form at the specified concentration using nuclease-free, MilliQ or equivalent molecular biology grade water.

0.9% Saline (154 mM) may be prepared by diluting 15.4 mL of 5 M NaCl in MilliQ or equivalent molecular biology grade water to a final volume of 500 mL. Equivalent preparations or commercial saline products may be utilized, with appropriate validation.

A 100X Inactivation Solution is prepared by mixing the components in Table 4. Equivalent preparations utilizing components with different source concentrations may be used such that the final 100X Concentration is achieved. Aliquots of 100X Inactivation Solution should be stored in the dark at -20°C for up to 3 months. Upon thaw, working aliquots of 100X Inactivation Solution should be stored in the dark at room temperature for up to 1 month.

#### Table 4: 100X Inactivation Solution
| **Component**                 | **Source Concentration**  | **Volume for 100X** | **100X Concentration** |
| ------------------            | --------------------      | ---------------- | ------------------ |
| TCEP                          | 0.5 M                     | 10 mL            | 250 mM            |
| EDTA                          | 0.5 M                     | 4 mL             | 100 mM            |
| NaOH                          | 10 N                      | 2.3 mL           | 1.15 N            |
| Nuclease-free Water           |        -                  | 3.7 mL           |  -                |
| **TOTAL VOLUME**              |              -              | **20 mL**        |        -           |
||


For swabs that are collected or eluted in 0.9% saline solution or equivalent, the 100X Inactivation Solution should be added at 1/100th the sample solution volume.

For dry swabs, a preparation of 1X Inactivation Saline Solution should be prepared per Table 5. 1X Inactivation Saline Solution should be kept at room temperature and used within 24 hours of preparation from components or 100X Inactivation Solution.

#### Table 5: 1X Inactivation Saline Solution
|                                           |             |
|-------------------------------------------|-------------|
| **Component**                             | **Volume**  |
| 0.9% Saline (154 mM NaCl) in MilliQ Water | 1000 mL     |
| 100X Inactivation Solution                | 10 mL       |
| **TOTAL VOLUME**                          | **1010 mL** |
||

The **FloodLAMP QuickColor(TM) COVID-19 Test** uses 18 LAMP primers targeted for 3 different SARS-CoV-2 genes, with 6 primers for each target. Primer names and sequences are shown above in Table 1. All 18 primers are mixed together and input into a single amplification reaction.

Primers may be purchased from the vendor LGC Biosearch Technologies as 3 pre-blended sets, or the primers may be purchased as 18 individual custom oligos. Table 6 below lists the primer products to be ordered.

The LGC Biosearch primer products are provided already blended for each target (6 primers per tube) and dried such that upon resuspension with 1 mL of nuclease-free water, the primers for each target are at 30X concentration. One resuspended tube for each of the 3 targets (i.e. primer blends) are mixed together to yield a 3 mL total volume that contains all individual primers at 10X concentration. This 3 mL of 10X LAMP Primer Mix provides for 1,200 reactions at 2.5 µL per reaction.

Alternatively to the pre-blended LGC Biosearch products, primers may be purchased as individual custom oligos. Custom oligos may be blended to form 30X Primer Set Mixes as intermediates or all mixed together for the 10X LAMP Primer Mix. The FIP and BIP primers for each target require purification by HPLC or an equivalent process. Appropriate validation of primer mixes from custom oligos is required. Primers may be stored at 4°C for up to one month, or at -20°C for up to 1 year.

#### Table 6: 10X LAMP Primer Mix Components
| Vendor | Item | Catalog number | Quantity | # Reactions |
|--------|------|----------------|-----------|-------------|
| Order one of the following primer sets |||||
| LGC Biosearch Technologies | SARS-CoV-2 LAMP AS1e 6 primer set at 30X (ORF1ab gene) | LAMP_S2-AS1e-48 | 6-48 nmol | 1,200 |
| LGC Biosearch Technologies | SARS-CoV-2 LAMP AS1e 6 primer set at 30X (ORF1ab gene) | LAMP_S2-AS1e-480 | 60-480 nmol | 12,000 |
| LGC Biosearch Technologies | SARS-CoV-2 LAMP N2 6 primer set at 30X (N gene) | LAMP_S2-N2-48 | 6-48 nmol | 1,200 |
| LGC Biosearch Technologies | SARS-CoV-2 LAMP N2 6 primer set at 30X (N gene) | LAMP_S2-N2-480 | 60-480 nmol | 12,000 |
| LGC Biosearch Technologies | SARS-CoV-2 LAMP E1 6 primer set at 30X (E gene) | LAMP_S2-E1-48 | 6-48 nmol | 1,200 |
| LGC Biosearch Technologies | SARS-CoV-2 LAMP E1 6 primer set at 30X (E gene) | LAMP_S2-E1-480 | 60-480 nmol | 12,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_FIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_BIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_F3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_B3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_LF | Custom Order | 250 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | Orflab_LB | Custom Order | 250 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_FIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_BIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_F3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_B3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_LF | Custom Order | 250 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | N2_LB | Custom Order | 250 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_FIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_BIP | Custom Order | 1,000 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_F3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_B3 | Custom Order | 125 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_LF | Custom Order | 250 nmol | 25,000 |
| LGC Biosearch Technologies, Eurofins Genomics, Integrated DNA Technologies, Sigma | E1_LB | Custom Order | 250 nmol | 25,000 |
||

The primers are mixed with guanidine hydrochloride to form an intermediate Primer-Guanidine Solution prior to combining with the sample and Colorimetric LAMP MM for the full amplification reaction. The components and volumes for the Primer-Guanidine Solution are listed in Table 7 and may be proportionally scaled for batch sizes of different numbers of reactions. The Primer-Guanidine Solution may be prepared in advance and should be stored at -20°C for up to 1 month.

#### Table 7: Primer-Guanidine Solution
| **Component**         | **Volume (1 reaction)** | **Volume (100 reactions)** |
|-----------------------|-------------------------|----------------------------|
| 10X LAMP Primer Mix   | 2.5 µL                 | 250 µL                     |
| Guanidine (400 mM)    | 2.5 µL                 | 250 µL                     |
| Nuclease-free Water   | 5.5 µL                 | 550 µL                     |
| **TOTAL VOLUME**      | **10.5 µL**            | **1050 µL**                |
||

The final Colorimetric LAMP Amplification Reaction components are listed in Table 8. PCR plates or strip tubes used for the amplification reactions should be maintained on ice or a cold block until less than 5 minutes before incubation on the heater. Reaction plates/strip tubes comprising the Colorimetric LAMP Amplification Reaction Solution may be prepared in advance, capped/sealed, and stored at -20°C for up to 1 day prior to addition of the sample. A heated plate sealer is recommended. Alternatively, a manually applied foil or optical seal may be used.

#### Table 8: Colorimetric LAMP Amplification Reaction
| **Component**         | **Volume (1 reaction)** | **Volume (100 reactions)** |
|-----------------------|-------------------------|----------------------------|
| Primer-Guanidine Solution | 10.5 µL             | 1050 µL                    |
| Colorimetric LAMP MM     | 12.5 µL             | 1250 µL                    |
| **SUBTOTAL VOLUME**      | **23 µL**           | **2300 µL**                |
| Sample                   | 2 µL               | -                          |
| **REACTION VOLUME**      | **25 µL**           | -                          |
||

### 4) Software Validation
No software is required for labs to run the **FloodLAMP QuickColor(TM) COVID-19 Test*.***

### 5) Testing Capabilities
The **FloodLAMP QuickColor(TM) COVID-19 Test** has been optimized for a robust, streamlined workflow and rapid turnaround time on results. The total time to perform the test is dependent upon the following factors: number of lab technicians, batch size of samples, and advance preparation of reaction mixes. **The minimum turnaround time is approximately 45 minutes.**

Since the **FloodLAMP QuickColor(TM) COVID-19 Test** does not require specialized instruments, it can be scaled up without the capital investment required for PCR machines or automated extraction. The number of tests capable of being performed per day by a laboratory is not constrained by PCR machines or extraction instruments. One technician can manually process approximately one plate (94 samples) per hour. This includes intake (debagging and barcode scanning), inactivation, and LAMP amplification. Automation can greatly increase throughput.

### 6) Reagent Stability
A stability test plan for the components of the **FloodLAMP QuickColor(TM) COVID-19 Test** will be developed during an interactive review. Briefly, the proposed study includes assessing all prepared solutions including: 100X Inactivation Solution, 1X Inactivation Saline Solution, 30X Primer Stock, 10X Primer Mix, Primer-Guanidine Solution, and the full Colorimetric LAMP Amplification Reaction Mix. Prepared solutions will be assessed both for long term storage stability (typically 1-3 months at -20°C) and short term storage stability prior to usage (typically hours to several days at room temperature, 4°C or -20°C).

The proposed study uses a contrived positive sample consisting of inactivated SARS-CoV-2 virus cell lysate (BEI NR-52287) spiked into negative specimens at approximately 50,000 copies/mL (4X LoD). The contrived positive stability study samples will be prepared, aliquoted and stored at -80°C to permit repeated testing of the various solutions at the appropriate step of the test protocol.

For test components supplied by vendors, such as the Colorimetric LAMP Master Mix, the manufacturer's recommended storage conditions and duration will be followed.

### 7) Sample Stability
Upper respiratory specimens including nasopharyngeal swabs, anterior nasal and mid-turbinate nasal swabs should be collected using standard procedures and recommendations. Swab specimens should be collected in 0.9% saline, PBS, or dry tubes. Specimens should not be collected in UTM, VTM, or Liquid Amies.

Please refer to Interim Guidelines for Collecting, Handling, and Testing Clinical Specimens for COVID-19: https://www.cdc.gov/coronavirus/2019-ncov/lab/guidelines-clinical-specimens.html

The stability study of the nasal swab sample transported in saline has been conducted by Quantigen Biosciences, with support from The Gates Foundation and UnitedHealth Group. Quantigen Biosciences has granted a right of reference to any sponsor wishing to pursue an EUA to leverage their COVID-19 swab stability data as part of that sponsor’s EUA request.

Samples can be stored at room temperature for 56 hours after collection prior to inactivation. For longer term storage, samples can be stored at ≤-70°C.

## J. PERFORMANCE EVALUATION
### 1) Limit of Detection (LoD) - Analytical Sensitivity:
The Limit of Detection (LoD) for the **FloodLAMP QuickColor(TM) COVID-19 Test** was established using gamma-irradiated SARS-CoV-2 virus cell lysate (BEI NR-52287) spiked into negative real specimens. The negative specimens were confirmed by PCR using the CDC primers. The gamma-irradiated virus was spiked into the specimen prior to the heat inactivation step, and carried through the entire assay. The concentration of spike was such that the contrived positive sample was at 100,000 copies/mL after the inactivation step. The stock contrived positive was diluted into inactivated negative sample matrix to produce the concentrations for the LoD study. A preliminary LoD run was performed using the concentrations ranging from 100,000 copies/mL to 3,100 copies/mL. Concentrations of 12,500 and 6,250 were selected for confirmatory LoD runs. LoD run details are provided in Supporting Data, with the results summarized below in Table 9. The LoD, defined as the concentration at which at least 95% of the samples are positive, was determined at 12,500 copies/mL.

#### Table 9: LoD Confirmatory Data Results
| **Concentration of Contrived Positive Sample** | **Replicates Detected** |
|-----------------------------------------------|-------------------------|
| 12,500 copies/mL                              | 95% (20/21)            |
| 6,250 copies/mL                               | 52% (11/21)            |
||

### 2) Inclusivity (analytical sensitivity):
An inclusivity study was conducted for the ORF1ab, N2, and E1 primer sets against all complete, high coverage SARS-CoV-2 sequences deposited at GISAID as of February 27, 2021. Table 10 summarizes the results of this in silico inclusivity analysis. A total of 498,224 sequences were considered. There are 10 sequence isolates that have 1mm to both As1e and E1 and had N2 excluded due to greater than 15 N's, with the other 498,214 sequence isolates all have at least 1 target region that is a complete match.

Each primer set matched at 100% similarity against the SARS-CoV-2 RefSeq reference genome (Wuhan-Hu-1; NC\_045512.1). All three primer sets differed by one or fewer mutations for 99.7% of GISAID sequences, indicating nominal primer hybridization for all SARS-CoV-2 variants under consideration.

#### Table 10: *In Silico* Inclusivity Analysis for LAMP Primers
| Primer | AS1e  (ORF1ab gene) |  | N2  (N gene) |  | E1  (E gene) |  |
| :---- | ----- | ----- | ----- | ----- | ----- | ----- |
| **Total Primer Length** | 195 |  | 169 |  | 168 |  |
| **Total Number of Strains Evaluated** | 498,224 |  | 498,224 |  | 498,224 |  |
| **100% Match** | 474,717 | 95.3% | 479,548 | 96.3% | 462,538 | 92.8% |
| **1 Mismatch** | 19,301 | 3.9% | 15,698 | 3.2% | 30,626 | 6.1% |
| **2 Mismatches** | 338 | 0.1% | 161 | 0.0% | 1,455 | 0.3% |
| **3 Mismatches** | 9 | 0.0% | 5 | 0.0% | 103 | 0.0% |
| **\> 3 Mismatches** | 0 | 0.0% | 0 | 0.0% | 1 | 0.0% |
| **Total Strains Removed** | 3,859 | 0.8% | 2,812 | 0.6% | 3,501 | 0.7% |
||

#### Evaluation of Impact of Viral Mutations
The As1e, E1 and N2 primer regions of all SARS-CoV-2 genomes present in GISAID as of 2/26/2021 were evaluated to assess the potential impact of genomic variants on LAMP primer binding. This analysis was performed with the Primer Monitoring Tool from New England Biolabs (<https://primer-monitor.neb.com/>), which continually monitors registered primer sets for overlapping variants in sequences from GISAID. Results are summarized by region and locus below in Table 11, including the 30 countries with most sequences in GISAID. Sequences were aligned to the SARS-CoV-2 reference sequence (NC\_045512.2) using minimap2 (minimap2 -t 16 -x asm5 -a). Variant sites (excluding Ns) were identified using samtools mpileup and summarized by region and genome position. Genomic positions having &gt;= 40 global variant observations are shown (column labels). When present, box labels indicate the fraction of variants observed at a given locus.

The aggregate of current published mutations is not expected to reduce performance of the **FloodLAMP QuickColor(TM) COVID-19 Test** by more than 5% from that established by the performance evaluation in this EUA submission. Further, the use of 3 primer sets targeting different regions in the SARS-CoV-2 genome should make the test robust to new genetic variants.

#### Table 11: Variant Analysis of LAMP Primers
![][image3]  
![][image4]

### 3) Cross-reactivity (Analytical Specificity):
*In silico* cross-reactivity analysis was performed by aligning the primer sequences of the **FloodLAMP QuickColor(TM) COVID-19 Test** against sequences of other coronaviruses and common respiratory flora using the BLASTn alignment tool from NCBI. Results of this analysis are presented in Tables 12A, 12B, and 12C.

The % identity range (# identical bases/ \# primer bases) is shown for each primer and organism. Darker font indicates % identity greater than 80%. Organisms with &gt;= 50% identity primer hits are shown. This analysis is not intended to predict amplification. Near perfect homology across B3, F3, FIP and BIP is necessary to support successful amplification. With the exception of SARS-CoV, simultaneous homologies do not occur between any of the primers and microorganisms screened. With respect to clinical relevance of the in silico cross-reactivity analysis, there are no known circulating strains of SARS-CoV circulating in humans, thus the overall probability for the test to produce a cross-reactive signal is negligible.

#### Table 12A: *In Silico* Cross-Reactivity Analysis for AS1e Primers
![][image5]

#### Table 12B: *In Silico* Cross-Reactivity Analysis for N2 Primers
![][image6]

#### Table 12C: *In Silico* Cross-Reactivity Analysis for E1 Primers
![][image7]

Wet testing was performed to demonstrate that the **FloodLAMP QuickColor(TM) COVID-19 Test** does not react with related pathogens, high prevalence disease agents and normal or pathogenic flora that are reasonably likely to be encountered in a clinical specimen. SARS-CoV, RSV, Flu, Human Metapneumovirus. and Streptococcus Salivarius were tested for potential cross-reactivity, as shown in Table 13 and Supporting Data. 5 µL of each stock of cross-reactivity organism was spiked on dried AN swab specimens. A contrived positive control was produced by spiking gamma-irradiated SARS-CoV-2 virus cell lysate (BEI NR-52287) onto dried AN swab specimens. Control dried swabs obtained simultaneously were confirmed to be SARS-CoV-2 negative by PCR using the CDC primers. The gamma-irradiated SARS-CoV-2 virus and cross-reactivity organisms were spiked onto the dried swabs prior to the heat inactivation step, and carried through the full test protocol. The contrived positive had 38 µL of 1e6 copies/mL irradiated virus stock spiked in, producing after elution of the swab in 1 mL of Inactivation Saline Solution at most a concentration of 38,000 copies/mL in the sample input into the amplification reaction.

All wet testing showed no cross-reactivity with the viral pathogens and common respiratory flora, as shown in Table 13.

#### Table 13: Wet Testing Cross-Reactivity Results
| **Organism**           | **Description**         | **BEI Number** | **Detected Replicates** |
|------------------------|-------------------------|---------------|-------------------------|
| SARS-CoV               | UV-inactivated virus    | NR-3882       | 0/3                     |
| Human Metapneumovirus  | Genomic RNA            | NR-49122      | 0/3                     |
| RSV                    | Genomic RNA            | NR-43976      | 0/3                     |
| Influenza B            | Genomic RNA            | NR-45848      | 0/3                     |
| Streptococcus salivarius | Bacterial cell culture | HM-121        | 0/3                     |
||

#### Endogenous Interference Substances Studies:
Exogenous and endogenous substances were tested for potential interference with the **FloodLAMP QuickColor(TM) COVID-19 Test.** 10 µL of each stock of interfering substance was spiked on dried AN swab specimens. A contrived positive control was produced by spiking gamma-irradiated SARS-CoV-2 virus cell lysate (BEI NR-52287) onto dried AN swab specimens. Control dried swabs obtained simultaneously were confirmed to be SARS-CoV-2 negative by PCR using the CDC primers. The gamma-irradiated SARS-CoV-2 virus and interfering substances were spiked into the dried swabs prior to the heat inactivation step, and carried through the full test protocol. The contrived Positive Control Spiked comprised 20 µL of 8e6 copies/mL irradiated virus stock spiked in, producing after elution of the swab in 1 mL of Inactivation Saline Solution at most a concentration of 160,000 copies/mL in the sample input into the amplification reaction.

All interfering substance testing showed no disagreement with expected positive and negative results, as shown in Table 14 and Supporting Data.

#### Table 14: Interfering Substances Results
| **Interfering Substance** | **Active Ingredient**                        | **Concentration** | **% Agreement (Positive)** | **% Agreement (Negative)** |
|---------------------------|---------------------------------------------|-------------------|---------------------------|---------------------------|
| Blood                    | N/A                                         | 1% v/v           | 100% (3/3)               | 100% (3/3)               |
| Nasal Congestion Spray   | Acetaminophen,<br>Guaifenesin,<br>Phenylephrine HCI | 20% v/v          | 100% (3/3)               | 100% (3/3)               |
| Nasal Allergy Spray      | Oxymetazoline HCl                           | 15% v/v          | 100% (3/3)               | 100% (3/3)               |
| Lozenges                 | Menthol                                     | 10% w/v          | 100% (3/3)               | 100% (3/3)               |
| Mucin                    | N/A                                         | 0.5% w/v         | 100% (3/3)               | 100% (3/3)               |
||

### 4) Clinical Evaluation
The clinical evaluation of the **FloodLAMP QuickColor(TM) COVID-19 Test** utilized confirmed clinical anterior nares swab specimens. 40 positive and 40 negative clinical specimens were evaluated and compared to a high sensitivity EUA authorized test run on the original fresh samples. The **FloodLAMP QuickColor(TM) COVID-19 Test** showed a positive agreement of 90% and a negative agreement of 100%. The 4 false negative results were specimens with high Ct values as previously measured by the comparator test, indicating low viral load. A summary of the clinical performance is below in Table 15.

Anterior nares swab specimens were collected from patients in phosphate buffered saline by the Stanford COVID-19 clinical testing program. Specimens were initially tested by the Stanford clinical laboratory using the Hologic Panther Fusion and Aptima SARS-CoV-2 Assays, which serves as the high sensitivity comparator test.

For the **FloodLAMP QuickColor(TM) COVID-19 Test**, materials and the Instructions For Use were provided to the Stanford clinical laboratory. The materials provided consisted of the validated reagents listed in Table 3, the LGC primers and probes, and an aliquot of the positive control. After thawing the frozen specimens, 1 mL of each specimen was transferred to 5mL tubes for the inactivation step. The positive and negative clinical specimens were assigned a new ID in a random order, then transferred to new tubes that were barcoded and labeled with the new ID. The Bio-Rad C1000 Touch(TM) thermal cycler was used for the heating device to perform the isothermal amplification. Two different technicians independently interpreted the results visually per the Instructions For Use, with identical results. Line Item data are provided in the Supporting Data.

Of the 40 positive specimens, 7 specimens had initial inconclusive results due to color variation beyond the edge case examples. Per Section H2 above, the inactivated samples were follow-up tested with the FloodLAMP EasyPCR(TM) COVID-19 Test, for which all 7 inconclusive results were positive.

#### Table 15: Clinical Evaluation Results
|                                    | **Comparator - High Sensitivity EUA Authorized Test** |     |     |
|------------------------------------|--------------------------------------------------------|-----|-----|
| **FloodLAMP QuickColor(TM) COVID-19 Test Results** | **Positive**                                           | **Negative** | **Total** |
| Positive                           | 36                                                     | 0           | 36        |
| Negative                           | 4                                                      | 40          | 44        |
| Total                              | 40                                                     | 40          | 80        |
| Positive Agreement                 | 90.0% (36/40) 95% CI: 76.3% to 97.2%                   |             |           |
| Negative Agreement                 | 100% (40/40) 95% CI: 91.2% to 100%                     |             |           |
||

## K. UNMET NEED ADDRESSED BY THE PRODUCT
This section will be completed by FDA.

## L. APPROVED/CLEARED ALTERNATIVE PRODUCTS
Currently no methods for the detection of the SARS-CoV-2 have been approved/cleared by FDA.

## M. BENEFITS AND RISKS:
This section will be completed by FDA.

## N. FACT SHEET FOR HEALTHCARE PROVIDERS AND PATIENTS:
Fact Sheets for Patients and Healthcare Providers attached.

## O. INSTRUCTIONS FOR USE/ PROPOSED LABELING/PACKAGE INSERT:
Instructions for Use attached.

## P. RECORD KEEPING AND REPORTING INFORMATION TO FDA:
Authorized laboratories will collect information on the performance of the test and report to DMD/OHT7-OIR/OPEQ/CDRH (via email: CDRH-EUA-Reporting@fda.hhs.gov) and FloodLAMP Biotechnologies, PBC support center (via email: eua.support@floodlamp.bio) any suspected occurrence of false positive or false negative results and significant deviations from the established performance characteristics of the test of which they become aware.
